src/assets/lang/en/Send.json

```diff
@@ -0,0 +1,73 @@
+{
+  "FeatureSection": {
+    "title": "Secure from the moment you hit send",
+    "description": "Files uploaded and sent through Internxt Send use Internxt's disruptive zero-knowledge architecture to maximize the security and privacy of files. Maintain rock-solid confidentiality and safely transmit sensitive information with Internxt Send.",
+    "card1": {
+      "title": "User-first the sharing",
+      "description": "Internxt Send respects users' right to privacy and doesn't scan, store, or save any information you send through the service."
+    },
+    "card2": {
+      "title": "Private, encrypted file transfer",
+      "description": "All files and photos are end-to-end encrypted, meaning all your information is scrambled and fragmented before even leaving your device."
+    },
+    "card3": {
+      "title": "Send large files in secrecy",
+      "description": "Send and share files no matter the size. Internxt Send has no file size limits and can send large files in no time."
+    },
+    "card4": {
+      "title": "Share files and photos of all formats",
+      "description": "All file formats are supported and can easily be encrypted and shared with Internxt Send. PDF, JPEG, DOC, PNG, XLS, and many more."
```

Comment:
the comma after XLS should be removed:
`XLS, and many more` instead do:
`XLS and many more`